src/site/stages/build/process-cms-exports/filters.js

```diff
@@ -5,7 +5,7 @@ const { getContentModelType } = require('./helpers');
  */
 
 const whitelists = {
-  global: ['title'],
+  global: ['title', 'baseType', 'contentModelType'],
```

Comment:
We need these in the transformed entities.

...:thinking: Or at least `contentModelType`...